configure

```diff
@@ -1,13 +1,17 @@
 #!/usr/bin/env python
 
+import sys
+if sys.version_info >= (3, 0):
+  sys.stdout.write("Python 3.x is not compatible at this time\n")
```

Comment:
@jbergstroem true, but I don't see a downside to not having. If we suggest the `python2` command there's a chance it won't work, and we'd be better off not suggesting a command in that case.